### PR TITLE
Transition Autopilot from the cloud overview page

### PR DESCRIPTION
**Pull request by anthonydillon**
## Done
Update the cloud overview page to replace autopilot with conjure-up.

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/cloud](http://0.0.0.0:8001/cloud)
- Check that the content matches the [copy doc](https://docs.google.com/document/d/1mQjVXDcoP8ChOzRdmuI55BdXhbURD53qBNc2bgJhfhg/edit#)
- Check on different screen sizes

## Issue / Card
Fixes https://github.com/canonical-websites/www.ubuntu.com/issues/2187

### CODE REVIEW

File: templates/cloud/index.html [hunk 1]
```diff
@@ -92,56 +120,122 @@
...
   </div>
 </section>
 
-<section class="p-strip--dark is-deep">
+<section class="p-strip--image u-no-background--small is-deep is-bordered" style="background-image: url('https://assets.ubuntu.com/v1/1d2ab6ba-suru-background.png'); background-size: 100% 100%;">
+  <div class="row">
+    <div class="col-8">
+      <h2>Foundation Cloud Build for Ubuntu OpenStack</h2>
+      <p>We will build you a highly available production cloud, implemented on-site with Canonical&rsquo;s expert delivery team in the shortest possible time.</p>
+      <p>Foundation Cloud Build makes redeploying or cloning your cloud architecture easy and affordable. We will also handle any important changes to your infrastructure including config modifications, OS and platform version upgrades, and deeper design revisions.</p>
```
Comment (deadlight) [inline templates/cloud/index.html]: Content here doesn't match the copydoc

Comment (anthonydillon) [reply templates/cloud/index.html]: Done, thanks.

Comment (pmahnke) [reply templates/cloud/index.html]: this is still missing the CTA [Find out more about Foundation Cloud Build >](https://www.ubuntu.com/cloud/foundation-cloud)

Comment (anthonydillon) [reply templates/cloud/index.html]: Good spot, thanks. Fixed

**Comment by anthonydillon** [comment]
@pmahnke ready for another review.

**Comment by kwm14** [comment]
## Visual review

### Also install Kubernetes with conjure-up
- [ ] missing the first line from the copy doc 'The Canonical Distribution of Kubernetes is a pure Kubernetes brought to you by the people who deliver'

### Fully managed OpenStack cloud
- [ ] missing logo - Fibernet - https://assets.ubuntu.com/v1/2353fc84-fibernet.svg
- [ ] missing logo - Cogeco Datacenters - https://assets.ubuntu.com/v1/97a050d0-Groupe_Cogeco_logo.svg

### Landscape
- [ ] All content doesn't match what is seen in [copy doc](https://docs.google.com/document/d/1mQjVXDcoP8ChOzRdmuI55BdXhbURD53qBNc2bgJhfhg/edit#heading=h.4a4653vo58j)

**Comment by anthonydillon** [comment]
@kwm14 Cheers, all fixed up. Ready for another review.

**Pull request merged**